src/components/organisms/course-preview/course-preview.tsx

```diff
@@ -62,19 +62,21 @@ export const CoursePreview: FC<CoursePreviewProps> = ({
 
           <Tag className={styles.level} text={level} />
 
-          {duration && (
+          {duration && duration > 0 && (
```

Comment:
Можно просто проверить на falsy-значение, 0 тоже в него входит

```suggestion
          {duration && (
```